src/main/java/net/masterthought/cucumber/ReportInformation.java

```diff
@@ -1,11 +1,6 @@
 package net.masterthought.cucumber;
 
-import java.text.SimpleDateFormat;
-import java.util.ArrayList;
-import java.util.Date;
-import java.util.List;
-import java.util.Map;
-
+import com.googlecode.totallylazy.Sequence;
```

Comment:
It not about the order but about potential conflicts that may happen if you add changes that can be avoided